test/client_bandit_test.py

```diff
@@ -98,12 +106,45 @@ def test_get_bandit_action_bandit_does_not_exist():
     assert result == BanditResult("default_variation", None)
 
 
-def test_get_bandit_action_flag_without_bandit():
+def test_get_bandit_action_flag_has_no_bandit():
```

Comment:
have this use a real flag and get assigned a non-default (but non-bandit) variation